pkgs/applications/office/planner/default.nix

```diff
@@ -1,69 +1,18 @@
-{ stdenv
-, fetchgit
-, pkgconfig
-, intltool
-, automake111x
-, autoconf
-, libtool
-, gnome2
-, libxslt
-, python
-}:
+{ newScope, stdenv }:
 
-let version = "20170425";
-
-in stdenv.mkDerivation {
-  name = "planner-${version}";
-
-  src = fetchgit {
-    url = "https://git.gnome.org/browse/planner";
-    rev = "6a79647e5711b2b8d7435cacc3452e643d2f05e6";
-    sha256 = "18k40s0f665qclrzvkgyfqmvjk0nqdc8aj3m8n4ky85di4qbqlwd";
-  };
-
-  buildInputs = with gnome2; [
-    pkgconfig
-    intltool
-    automake111x
-    autoconf
-    libtool
-
-    gnome_common
-    gtk_doc
-
-    GConf
-    gtk
-    libgnomecanvas
-    libgnomeui
-    libglade
-    scrollkeeper
-
-    libxslt
-    python
-  ];
-
-  preConfigure = ''./autogen.sh'';
-
-  enableParallelBuilding = true;
+let
+  callPackage = newScope stdenv;
```

Comment:
Is there a specific reason for using this over the simpler `import ./foo.nix { inherit stdenv; ... }`?